lib/app_home.dart

```diff
@@ -40,6 +43,51 @@ class AppHome extends StatefulWidget {
 }
 
 class _AppHomeState extends State<AppHome> {
+  @override
+  void initState() {
+    super.initState();
+    SplashWindowCloseNotifier.setWindowCloseHandler(
+      onClose: () async {
+        return await showDialog(
```

Comment:
unnecessary await & async. the future can be returned without awaiting here
```
onClose: () {
  return show...
```

Reply:
Fixed with the method extraction.